src/script/components/AvailabilityState.tsx

```diff
@@ -27,6 +27,7 @@ import NamedIcon from './NamedIcon';
 
 export interface AvailabilityStateProps {
   availability: Availability.Type;
+  className: string;
```

Comment:
Is this optional? Because you have a `if (className)` check later on...

Reply:
https://github.com/wireapp/wire-webapp/pull/10469/commits/e8aea880f2beff8c6d8c8426bf4a1ec6cf26f318